dev/devicelab/manifest.yaml

```diff
@@ -702,6 +702,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/ios"]
 
+  tiles_scroll_perf_macos__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on macOS.
+    stage: devicelab
+    required_agent_capabilities: ["mac/ios"]
```

Comment:
FYI @godofredoc @stuartmorgan  , we probably need a mac specific config in the devicelab. If we're planning on using VMs for the desktop machines, then we probably shouldn't start testing them on physical devices since we'll have to rebaseline?

Reply:
I suggest to try to get them running on LUCI bots instead of the devicelab.

Reply:
Are there any performance benchmarks that are run on LUCI bots today? I am thinking about how the results should then be posted to skia-perf. cc: @liyuqian

Reply:
Rebaseline wouldn't be an issue for Skia perf. We'll see a jump when we migrate the machines, but we (Ray) can simply mark all those jumps as green in one button tap.

I guess the current blocker of using LUCI is how to report metrics. I plan to solve that in Q4. We can have a few desktop benchmarks on devicelab before that as long as we don't deplete device lab resources with too many desktop benchmarks.

Reply:
> Rebaseline wouldn't be an issue for Skia perf. We'll see a jump when we migrate the machines, but we (Ray) can simply mark all those jumps as green in one button tap.
> 
> I guess the current blocker of using LUCI is how to report metrics. I plan to solve that in Q4. We can have a few desktop benchmarks on devicelab before that as long as we don't deplete device lab resources with too many desktop benchmarks.

Is there any way the we can reprioritize the metrics uploading part? From what I've seen we only need to create a skia perf client that sends a request to skia perf apis with the benchmarks collected and authenticating with a token provided by luci.

Once this client is available as a cipd package the runner can use an env variable to identify the skia perf client is available and call it. This integration will be very similar to the skia golds integration.

Reply:
As I see it, if we were to run these test on LUCI bots we would still need to:
1. Add support for perf tests to be run on macOS
2. Add a task that can be triggered via `bin/run.dart -t`

This PR additionally also does:
3. Trigger it in existing device lab (`manifest.yaml` changes).

Given that re-baselining is low lift and moving these to LUCI bots would require additional work that is slated for Q4, is there any opposition to having these tests run in devicelab for this quarter? I would expect the mac hosts to be running under capacity right now as all the existing tests are likely being run on iOS devices.

Reply:
There is not a separate mac pool

Reply:
I guess Kaushik's question is whether we can use the "mac/ios" pool to run a few desktop benchmarks this quarter before LUCI is ready. That seems fine to me unless there's some red flag in the devicelab saying that our mac/ios pool is currently under big pressure.

Reply:
it will too easy to let the long term solution slide if we accept more short term infra hacks.

I would prefer that adding the required support to LUCI was prioritized over short term fixes

Reply:
> it will too easy to let the long term solution slide if we accept more short term infra hacks.
> 
> I would prefer that adding the required support to LUCI was prioritized over short term fixes

In addition to that, the devicelab queue time is ~3 hours and we are already migrating some parts of it to LUCI.